src/test/scala/io/iohk/ethereum/jsonrpc/JsonRpcControllerSpec.scala

```diff
@@ -1307,7 +1304,7 @@ class JsonRpcControllerSpec extends FlatSpec with Matchers with PropertyChecks w
       override val blockCacheSize: Int = 30
       override val ommersPoolSize: Int = 30
       override val txPoolSize: Int = 30
-      override val poolingServicesTimeout: FiniteDuration = 3.seconds
+      override val poolingServicesTimeout: FiniteDuration = Timeouts.normalTimeout
     }
 
     val filterConfig = new FilterConfig {
```

Comment:
Minor detail: Should we use some of the timeouts from `Timeouts` for all the timeouts in this `FilterConfig`?